### DISCUSSION TITLE
Stale issues

### DISCUSSION THREAD

**Discussion by daghsentinel** [bagetter/BaGetter#35] [Ideas]
Is there a smart way to automatically close stale issues due to inactivity?

**Comment by seriouz** [comment]
@daghsentinel Yes as GitHub action https://github.com/marketplace/actions/close-stale-issues

**Comment by daghsentinel** [comment]
Should we consider adding a threshold/stale action, perhaps for as long as 3 months as this is a low volume project?

**Reply by seriouz** [reply]
Yeah I think 3 month should be suitable for the moment.

**Comment by daghsentinel** [comment]
I tried to create a workflow with stale issue action in it, but failed on creating a fine grained token for this repo. The repo never show up in the list of available repos I can create a token for. Any ideas? Too limited access?

**Reply by seriouz** [reply]
I had not problem adding the Action. Did you commit the .yml to the main branch like so? https://github.com/bagetter/BaGetter/blob/main/.github/workflows/stale.yml

**Reply by daghsentinel** [reply]
No, I didn't get that far. I tried to create the github token first. I am not used to actions at all. Does the `secrets.GITHUB_TOKEN` token exist in the secret list as some sor of default secret?

**Reply by seriouz** [reply]
Yes it's an default internal secret of the repo.

**Reply by daghsentinel** [reply]
Always learning something new. Thx.

**Comment by daghsentinel** [comment]
Resolved in https://github.com/bagetter/BaGetter/commit/c7614f5b773317efe026beed17128035a05c0f12
